tests/integration_tests/cli_dummy_integration.sh

```diff
@@ -0,0 +1,15 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+#!/bin/bash
+
+echo "I ran a dummy integration test."
```

Comment:
Is this intentionally checked in?

Reply:
These are used so we don't actually run a full integration_test. Just to make sure that the commands discover the appropriate test and can run them.

Reply:
I'll add a comment to this file as well.